src/plugins/intel_cpu/CMakeLists.txt

```diff
@@ -277,6 +277,30 @@ target_include_directories(${TARGET_NAME} SYSTEM PRIVATE $<TARGET_PROPERTY:dnnl,
 # is not (yet) needed.
 target_include_directories(${TARGET_NAME} PRIVATE $<TARGET_PROPERTY:openvino::reference,INTERFACE_INCLUDE_DIRECTORIES>)
 
+# Check if SVE is available for AARCH64; compile with that if yes.
+if (ARM OR AARCH64)
+    execute_process(
+        COMMAND lscpu
+        OUTPUT_VARIABLE CPUINFO
+        OUTPUT_STRIP_TRAILING_WHITESPACE
+    )
```

Comment:
It's not correct to detect capabilities of **build** processor during the build process, because it's quite typical that you compile even on x86 processor and then run on arm64

I think here you need to detect compiler capabilities and if it supports SVE instrincis, then HAVE_SVE is available

Reply:
In my latest commit I've removed this section and used `__ARM_FEATURE_SVE` to detect compiler capability instead.

My rationale of doing it the previous way was to account for situations where the compiler has SVE support (e.g., GCC 10.2+) but the hardware doesn't (e.g., Graviton2). In those cases, I think the SVE code will be compiled but will cause issues at runtime due to the hardware not being able to execute it. Is my understanding incorrect?

Reply:
> In those cases, I think the SVE code will be compiled but will cause issues at runtime due to the hardware not being able to execute it. Is my understanding incorrect?

Your understanding is correct, that is why I've written in another comment that you need to implement `with_cpu_sve` which performs runtime check and calls SVE code only if it's supported by processor.